float32 x
float32 y
float32 confidence

float32 area
float32 circularity

